docs/user-manual.md

```diff
@@ -0,0 +1,100 @@
+
+# Cambridge Particle Tracks - User's manual
+
+<!-- TOC start -->
+
+- [Introduction](#introduction)
+- [Usage](#usage)
+    * [Starting the tool](#start)
+    * [Loading the data](#load)
+    * [Adding a new particle](#newparticle)
+    * [Measuring particle properties](#measure)
+        + [Decay length](#length)
+        + [Radius of curvature](#radius)
+        + [Decay angles](#angles)
+        + [Stereoshift](#stereoshift)
+    * [Measuring the image magnification](#magnification)
+    * [Saving the data](#save)
+- [Useful keyboard shortcuts](#tips)
+- [References](#references)
+
+
+<!-- TOC end -->
+
+<!-- TOC --><a name="introduction"></a>
+## Introduction
+The Cambridge Particle Tracks (CPT) tool has been developed for the analysis of particle tracks in a detector. The tool is implemented as a [napari] plugin. [napari] is an open-source, multi-dimensional image viewer for Python, which allows the visualisation and analysis of multi-dimensional data. Our application is relatively simple, as we only use it to visualise 2D images, and we are not making use of the full capabilities of the library. To that end, the CPT plugin provides a simple and (hopefully) intuitive interface to interact with the data.
+
+The CPT tool is designed to be used with the data from CERN bubble chamber experiments that have been digitised and stored in the PartII Particle Tracks lab[^1]. The software allows the user to load and visualise the data, and to perform some basic analysis on the tracks. This manual describes how to use the tool to perform the measurements required for the practical.
+
+
+<!-- TOC --><a name="usage"></a>
+## Usage
+
+<!-- TOC --><a name="start"></a>
+### Starting the tool
+
+To start the tool, double-click on the CPT icon. This will open the `napari` window. To load the CPT plugin, click on the `Plugins` menu and select `cavendish-particle-tracks`. This will the CPT window, which will be empty until you load a data file.
```

Comment:
```suggestion
To start the tool, double-click on the CPT icon. This will open the `napari` window. To load the CPT plugin, click on the `Plugins` menu and select `cavendish-particle-tracks`. This will open the CPT window, which will be empty until you load a data file.
```